cmd/influx/pkg.go

```diff
@@ -138,7 +136,7 @@ func (b *cmdPkgBuilder) pkgApplyRunEFn() func(*cobra.Command, []string) error {
 			return err
 		}
 
-		_, diff, err := svc.DryRun(context.Background(), influxOrgID, pkg)
+		_, diff, err := svc.DryRun(context.Background(), influxOrgID, 0, pkg)
```

Comment:
userID set by http client, ignored here, is forced by telegraf/endpoints needing user ID locally in server